.travis.yml

```diff
@@ -1,17 +1,23 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
+    - 5.3
+    - 5.4
+    - 5.5
+    - 5.6
+    - hhvm
+
+env:
+    - SYMFONY_VERSION="2.3.*"
+    - SYMFONY_VERSION="2.4.*"
+    - SYMFONY_VERSION="2.5.*"
+    - SYMFONY_VERSION="2.6.*"
```

Comment:
With this configuration you get a really large build matrix (each PHP version is built with each Symfony version you specify). It were better to only build with different Symfony versions with a particular PHP version (for example 5.6) like this:

```yaml
matrix:
    include:
        - php: 5.3
        - php: 5.4
        - php: 5.5
        - php: 5.6
          env: SYMFONY_VERSION="2.3.*"
        - php: 5.6
          env: SYMFONY_VERSION="2.5.*"
        - php: 5.6
          env: SYMFONY_VERSION="2.6.*"
        - php: hhvm
```

By the way, Symfony 2.4 reached end of maintenance last year. Not sure if it's really worth to still test against it.